modules/ROOT/pages/amazon/amazon-sqs-connector-examples.adoc

```diff
@@ -5,41 +5,47 @@ endif::[]
+//  LIST ITEMS SHOULD BE ONE SENTENCE MAX. Additional statements, if necessary, belong in a paragraph beneath the item.  GLOBAL.  //
+. The second flow receives the message and logs the message body.
+
+This example uses variables for some field values. You can either replace the variables with actual values in the code, or you can provide the actual values for each variable in the `src/main/resources/mule-artifact.properties` file.
 
-Before you try the example, access Anypoint Studio, and check whether the Mule Palette displays entries for Amazon SNS and Amazon SQS. If not, follow the instructions in xref:amazon/amazon-sqs-connector-studio.adoc[Amazon SQS Studio Configuration] to install both connectors.
+Before you try the example, access Anypoint Studio and verify that Mule Palette displays entries for Amazon SNS and Amazon SQS. If not, follow the instructions in xref:amazon/amazon-sqs-connector-studio.adoc[Amazon SQS Studio Configuration] to install both connectors before trying this example.
+//  BECAUSE THIS IS A PREREQUISTE TO THE ENTIRE EXAMPLE, consider moving this text to the very beginning of this introduction, or creating an actual Prerequisites subheading.  //
 
 == Create a Flow to Send a Message
 
-Start the first flow by sending a message to the queue:
+Start the example by creating a flow that sends a message to the queue. Creating the flow involves creating a new Mule project and configuring the Listener operation, adding a transform message, adding and configuring an SQS Send Message operation, adding a Logger component to display the response, obtaining the number of messages in the queue, and adding a Logger to display that number.
+//  I SUGGEST ADDING THIS TEXT TO PREPARE THE AUDIENCE FOR THE SUBTOPICS WITHIN THIS CONTAINER TOPIC. This is one of the GLOBAL requirements for a topic set intro text, along with addressing the audience as "you" and telling "you" exactly WHY they want or need the information in this topic set.  //
+//  The following steps really should have its own subtitle, because it is not the entire process identified by the title preceding it.  //
 
 . Create a new Mule project in Anypoint Studio.
-. In the Mule Palette, search for *HTTP*, and select the *Listener* operation:
+. In Mule Palette, search for *HTTP* and select the *Listener* operation:
 +
 image:amazon/amazon-sqs-select-listener.png[select-listener]
 +
 . Drag the *Listener* operation onto the canvas.
 . In the *Listener* configuration, click *+* next to the *Connector configuration* field to add a global element.
 . Accept the defaults.
-. Set the *Path* field to `/`.
+. Set the *Path* field to `/`:
 +
 image::amazon/amazon-sqs-http-props.png[http-properties]
 
 === Add a Transform Message Component to Attach the Metadata
+// IS THERE ANY SORT OF TRANSITION SNIPPET THAT WOULD ADD VALUE here and serve as a leadin to the first step?  //
```

Comment:
```suggestion
```

Reply:
IS THERE ANY SORT OF TRANSITION SNIPPET THAT WOULD ADD VALUE here and serve as a leadin to the first step?

Reply:
I don't have time to address the transitions now since it would require a new tech review but I will note them in a JIRA